app/models/manageiq/providers/physical_infra_manager.rb

```diff
@@ -2,7 +2,10 @@ module ManageIQ::Providers
   class PhysicalInfraManager < BaseManager
     include SupportsFeatureMixin
 
+    has_many :switches, :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
```

Comment:
@Fryguy didn't want to have two providers with the same named association [InfraManager.switches](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/infra_manager.rb#L16) actually representing two different things, can you name this something like `physical_switches` and add a `:class_name => "Switch"`?

Reply:
Yes, I think I can do something like that. I am preparing some changes according to the comment below. I will do that change and will update this thread.

Reply:
Done.